Services/Administration/classes/class.ilAdministrationGUI.php

```diff
@@ -207,14 +207,28 @@ function &executeCommand()
 						}
 						else
 						{
-							if(is_subclass_of($class_name, "ilObject2GUI"))
+							if (!$this->creation_mode)
 							{
-								$this->gui_obj = new $class_name($this->cur_ref_id, ilObject2GUI::REPOSITORY_NODE_ID);
+								if(is_subclass_of($class_name, "ilObject2GUI"))
+								{
+									$this->gui_obj = new $class_name($this->cur_ref_id, ilObject2GUI::REPOSITORY_NODE_ID);
+								}
+								else
+								{
+									$this->gui_obj = new $class_name("", $this->cur_ref_id, true, false);
+								}
 							}
 							else
 							{
-								$this->gui_obj = new $class_name("", $this->cur_ref_id, true, false);
-							}						
+								if(is_subclass_of($class_name, "ilObject2GUI"))
+								{
+									$this->gui_obj = new $class_name(null, ilObject2GUI::REPOSITORY_NODE_ID, $this->cur_ref_id);
+								}
+								else
+								{
+									$this->gui_obj = new $class_name("", 0, true, false);
+								}
+							}
```

Comment:
Gehört das hierhin? Wie gesgt, die orgus sind bei uns doch uasserhalb der Administration gui...oder liegen die eigentlich noch in dem folder und werden nur ausserhalb angezeigt?